### PR TITLE
Add Plasma: command

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

File: lib/mogrify.ex [hunk 1]
```diff
@@ -166,6 +166,7 @@ defmodule Mogrify do
 
   defp normalize_arguments({"histogram:" <> option, nil}), do: ["histogram:#{option}"]
   defp normalize_arguments({"pango", params}), do: ["pango:#{params}"]
+  defp normalize_arguments({"plasma", params}), do: ["plasma:#{params}"]
```
Comment (kpanic) [inline lib/mogrify.ex]: Does it makes sense to add a couple of tests for this following what has been done for pango? https://github.com/route/mogrify/blob/master/test/mogrify_test.exs#L164

cc @talklittle  :wink:

Comment (ConnorRigby) [reply lib/mogrify.ex]: @kpanic i've added a couple tests

File: test/mogrify_test.exs [hunk 2]
```diff
@@ -160,6 +160,36 @@ defmodule MogrifyTest do
...
+
+  @tag :plasma
+  test "plasma binary output buffer matches file output" do
+    image =
+      %Image{}
+      |> custom("seed", 10)
+      |> custom("plasma", "fractal")
+
+    result1 = image |> custom("stdout", "png:-") |> create(buffer: true)
+    result2 = image |> create(path: Path.join(System.tmp_dir(), "1.png"))
```
Comment (kpanic) [inline test/mogrify_test.exs]: Style readability nit:

```elixir
image_from_stdout = 
  image 
  |> custom("stdout", "png:-") 
  |> create(buffer: true)

image_from_file = 
  image
  |> create(path: Path.join(System.tmp_dir(), "1.png")) 
```

File: test/mogrify_test.exs [hunk 3]
```diff
@@ -160,6 +160,36 @@ defmodule MogrifyTest do
...
+
+    buf1 = result1.buffer
```
Comment (kpanic) [inline test/mogrify_test.exs]: `buffer1 = image_from_stdout.buffer`

File: test/mogrify_test.exs [hunk 4]
```diff
@@ -160,6 +160,36 @@ defmodule MogrifyTest do
...
+    {:ok, buf2} = File.read(result2.path)
```
Comment (kpanic) [inline test/mogrify_test.exs]: `{:ok, buffer2} = File.read(image_from_file.path)`

File: test/mogrify_test.exs [hunk 5]
```diff
@@ -160,6 +160,36 @@ defmodule MogrifyTest do
...
+    assert buf1 == buf2
```
Comment (kpanic) [inline test/mogrify_test.exs]: `assert buffer1 == buffer2`

File: test/mogrify_test.exs (hunk 2 above)
Comment (talklittle) [reply test/mogrify_test.exs]: Don't think this style change is necessary as it's a copy-paste of the pango test.

Comment (kpanic) [reply test/mogrify_test.exs]: @talklittle I guess I had too much "style enthusiasm" :slightly_smiling_face: 

Thank you

**Comment by talklittle** [comment]
> @ConnorRigby It's the first time that I see the `plasma` feature!
> Would it make sense in this context to add something similar to what has been done for pango here?
> https://github.com/route/mogrify/blob/master/test/support/mogrify_detect.ex
> 
> In addition to this, would it makes sense to add an example to the README in this section? https://github.com/route/mogrify#examples

Agreed. I don't know what plasma is and if it's available on all platforms. Thanks for reviewing @kpanic

**Comment by ConnorRigby** [comment]
Sorry about this, but i accidentally deleted my fork of mogrify. I reopened this PR at #87

**Pull request closed without merging**